src/topkg_build.ml

```diff
@@ -33,11 +38,16 @@ let with_dir b dir = { b with dir }
 let nop = fun _ -> Ok ()
 
 let cmd c os files =
-  Topkg_os.Cmd.run @@ Topkg_cmd.(build_cmd c os %% of_list files)
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:false c os %% v "-just-plugin") >>= fun () ->
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:true c os %% of_list files)
```

Comment:
Just to make it clear. I don't like the PR in this current state because for now users can reuse [`Pkg.build_cmd`](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALbuild_cmd) and simply add arguments (like `-tag BLA` options according to the configuration) to it without them having to redefine everything. 

While this PR keeps backwards compat on this. Doing this after the PR, will break cross compilation for the package. So if possible the support for cross should go entirely in the definition of the `build_cmd`, if that is impossible we need to break the API and figure out a better way.